spec/experiment_spec.rb

```diff
@@ -119,6 +123,11 @@ def alternative(color)
       expect(experiment.resettable).to be_falsey
     end
 
+    it "should be possible to make an experiment retain user alternatives after reset" do
+      experiment = Split::Experiment.new("basket_text", :alternatives => ["Basket", "Cart"], :retain_user_alternatives_after_reset => true)
+      expect(experiment.resettable).to be_truthy
```

Comment:
Should this be `expect(experiment.retain_user_alternatives_after_reset).to be_truthy`?

Reply:
Oops, yes!